config/env/development.example.env

```diff
@@ -1,6 +1,7 @@
 # Application
-SECRET_KEY_BASE=""
-DEVISE_SECRET_KEY=""
+SECRET_KEY_BASE="7ba87bf80efaf54543aaa13f1a83cc598944c2c56411a6657c380156cdf2ef2950ffbbee5d3ae8ac07c65aaa12465b37ce5bbfc905f2adcc8ff18d481f29838d"
```

Comment:
Looks like you accidentally committed this! These values should be stored in `development.env`